### DISCUSSION TITLE
Home page.tsx updates not being displayed with additional deploys

### DISCUSSION THREAD

**Discussion by zthatch** [vercel/next.js#55262] [Help]
### Summary

My page.tsx updates are not being displayed. 

I have updated the returned section to have my portfolio entirely formatted in HTML as a first step, as can be seen here:  https://github.com/zthatch/nextjs-portfolio/blob/main/app/page.tsx. 

It runs locally with pnpm dev, and it builds successfully, and I have even switched the runtime on vercel from pnpm to bun to npm and back to pnpm by changing the lock files, but the very obviously changed page.tsx in my repo is still not displaying. Instead, it shows a much earlier version of this file.

I have tried re-deploying manually and not checking the cache button, but this did not work either. 

I am using the https://github.com/leerob/leerob.io template for deploying a basic portfolio. 

### Additional information

It may be relevant to mention that the page.tsx is stuck at a point prior to a force push where I reset to a previous commit.

**Comment by icyJoseph** [comment]
Is the correct branch setup in Vercel's dashboard?

Could you open a PR from another branch into your main branch, and see if that builds a correct preview? (The Vercel bot should post a link to the preview build)

**Reply by zthatch** [reply]
I've opened a PR where I add some text to the page.tsx, but the preview still seems to somehow being using my very old page.tsx.

https://nextjs-portfolio-git-test-branch-zthatch.vercel.app

**Reply by zthatch** [reply]
FYI: I have updated my original statement to include the fact that the page.tsx being rendered seems to be stuck at a point prior to a force push where I reset to a prior commit.

**Reply by icyJoseph** [reply]
Aha, maybe that has borked Vercel's logic? What if you create a new branch, say `release` and use that as your prod deploy branch?

Alternatively, I'd ask the Vercel support contact, if there's anything they can do about this.
